docs/docs/References/Workflows.md

```diff
@@ -32,6 +32,10 @@
 
 ![Screenshot](images/VirtualSourceSnapshot.png)
 
+## Virtual Source Create Empty VDB
+
+![Screenshot](images/VirtualSourceCreateEmpty.png)
```

Comment:
Could you share a link to the image you created for Create Empty VDB (maybe in your review description)?

Reply:
Added top-level comment to this PR